packages/authorization-process/src/services/authorizationService.ts

```diff
@@ -741,6 +745,34 @@ export function authorizationServiceBuilder(
 
       return { producerKeychain, showUsers: true };
     },
+    async getProducerKeychains({
+      filters,
+      authData,
+      offset,
+      limit,
+      logger,
+    }: {
+      filters: GetProducerKeychainsFilters;
+      authData: AuthData;
+      offset: number;
+      limit: number;
+      logger: Logger;
+    }): Promise<ListResult<ProducerKeychain>> {
+      logger.info(
+        `Retrieving producer keychains by name ${filters.name} , userIds ${filters.userIds}`
+      );
```

Comment:
`name` and `usersIds` are not the only filters we are using, so this log might be confusing. I know it's the same in `getClients` ([reference](https://github.com/pagopa/interop-be-monorepo/pull/563/files#diff-dbdfef4027ce7e4ab4a5697de6c66e5364588bf66e77ff0558e90d05e496dfd2R149)), I think it should be change there as well.
cc @rGregnanin